server/src/main/java/org/elasticsearch/action/delete/DeleteRequest.java

```diff
@@ -46,7 +46,8 @@
  * @see org.elasticsearch.client.Client#delete(DeleteRequest)
  * @see org.elasticsearch.client.Requests#deleteRequest(String)
  */
-public class DeleteRequest extends ReplicatedWriteRequest<DeleteRequest> implements DocWriteRequest<DeleteRequest>, CompositeIndicesRequest {
+public class DeleteRequest extends ReplicatedWriteRequest<DeleteRequest> implements DocWriteRequest<DeleteRequest>,
+    CompositeIndicesRequest {
```

Comment:
Could you add the new line before `implements` instead of here? Could you also indent this one more time so it doesn't line up with the class body?